types/context.go

```diff
@@ -146,61 +150,87 @@ func (c Context) multiStore() MultiStore {
 }
 
 // nolint
-func (c Context) BlockHeader() abci.Header {
-	return c.Value(contextKeyBlockHeader).(abci.Header)
-}
-func (c Context) BlockHeight() int64 {
-	return c.Value(contextKeyBlockHeight).(int64)
-}
+func (c Context) BlockHeader() abci.Header { return c.Value(contextKeyBlockHeader).(abci.Header) }
+
+// nolint
```

Comment:
I think you can just add a `// nolint` to the very top of the file to avoid having to duplicate.

Reply:
Done, thanks